src/parser/core/modules/Statistics/AbstractStatistic.tsx

```diff
@@ -5,6 +5,7 @@ export interface AbstractStatisticOptions {
 	info?: React.ReactNode
 	width?: ColumnSpan
 	height?: number
+	order?: number
```

Comment:
Prefer `displayOrder` for consistency with other modules.

Reply:
> Prefer `displayOrder` for consistency with other modules.

Can't comply.. these order values are a separate value. Statistic has it's own internal order and that name will conflict with the Statistic's panel's position on the page. I left the module-facing name to be displayOrder though, and just converted it to "statsDisplayOrder" somewhere in the middle and hopefully that won't cause confusion in the future.

Reply:
Would using statsPanelOrder consistently through the stack be a reasonable way to disambiguate this?  I'm not sure I want to block further on a naming debate, but wanted to raise a quick discussion to see if we can get a quick consensus here before I throw another one onto the tech debt pile.